api/constants/constants.go

```diff
@@ -414,3 +414,8 @@ const (
 	// the upgraded connection should be handled by the ALPN handler.
 	WebAPIConnUpgradeTypeALPN = "alpn"
 )
+
+const (
+	FileTransferRequest  string = "file-transfer-request@goteleport.com"
+	FileTransferResponse string = "file-transfer-request-response@goteleport.com"
+)
```

Comment:
I think I'll go with `Decision` since `Resolution` seems final (even tho more responses might be able to come in) and `Confirmation` to me seems too close to `Approve` but I'm probably over thinking. Anyway, `Decision` it is! Thanks for the brainstorm.

Reply:
Before I go and change everything, are we saying change every instance of `FileTransferResponse` to `FileTransferDecision` or just for these SSH requests